assets/sequences/sfxbanks/system.seq.inc

```diff
@@ -0,0 +1,1117 @@
+.table table_system
+entry system_menu_open
+entry system_menu_close
+entry system_puzzle_solved
+entry system_rupee
+entry system_timer_ding
+entry system_error
+entry system_error
+entry system_treasure_appear
```

Comment:
```suggestion
.table table_system
entry system_menu_open       # NA_SE_SY_WIN_OPEN
entry system_menu_close      # NA_SE_SY_WIN_CLOSE
entry system_puzzle_solved   # NA_SE_SY_CORRECT_CHIME
entry system_rupee           # NA_SE_SY_GET_RUPY
entry system_timer_ding      # NA_SE_SY_MESSAGE_WOMAN
entry system_error           # NA_SE_SY_MESSAGE_MAN
entry system_error           # NA_SE_SY_ERROR
entry system_treasure_appear # NA_SE_SY_TRE_BOX_APPEAR
```
This is more of an idea, but it would be really nice if the sfx enums were mapped to the table entries here so it can be much faster finding the sfx, and It'll be easier syncing the names once we find user-friendly names for all the sfx enum

Reply:
TBD. Could do.